teacher.rb

```diff
@@ -0,0 +1,15 @@
+require './person'
+
```

Comment:
- [OPTIONAL] I would advise you to use `require_relative` instead of `require` to import a ruby file inside another one. 

`require_relative` is a subset of `require` and is a convenient method to use when you are referring to a file that is relative to the current file you are working on **_(basically, within the same project directory)_**. In other words, you use `require_relative` to refer to the `person.rb` file since it is within the **_same directory (root directory in this case)_** as that of the `teacher.rb` file. 👍 😎